homeassistant/components/airos/config_flow.py

```diff
@@ -0,0 +1,118 @@
+"""Config flow for the Ubiquiti airOS integration."""
+
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from airos.airos8 import AirOS
+from airos.exceptions import (
+    ConnectionAuthenticationError,
+    ConnectionSetupError,
+    DataMissingError,
+    DeviceConnectionError,
+)
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+from homeassistant.const import CONF_HOST, CONF_PASSWORD, CONF_USERNAME
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_USERNAME, default="ubnt"): str,
+        vol.Required(CONF_PASSWORD): str,
+        vol.Required(CONF_HOST): str,
+    }
```

Comment:
I would put the `host` first I think

Reply:
Had that but decided to put the one with a default first, reverted